pkgs/development/compilers/llvm/common/libunwind/default.nix

```diff
@@ -60,15 +60,6 @@ stdenv.mkDerivation (finalAttrs: {
   ]
   ++ devExtraCmakeFlags;
 
-  prePatch = lib.optionalString (lib.versionOlder release_version "18") ''
```

Comment:
Doesn't the removal of `prePatch` and `postPatch` cause a rebuild?

Reply:
Yes. Will fix.